policy-controller/src/main.rs

```diff
@@ -9,9 +9,19 @@ use linkerd_policy_controller::{admin, admission};
 use linkerd_policy_controller_core::IpNet;
 use std::net::SocketAddr;
 use tokio::{sync::watch, time};
-use tracing::{debug, info, info_span, instrument, Instrument};
+use tracing::{debug, error, info, info_span, instrument, Instrument};
 use tracing_subscriber::{fmt::format, prelude::*, EnvFilter};
 
+use std::convert::Infallible;
+use std::{fs, io, sync};
+
+use async_stream::stream;
+use hyper::server::accept;
+use hyper::service::make_service_fn;
+use hyper::Server;
+use tokio::net::TcpListener;
+use tokio_rustls::TlsAcceptor;
```

Comment:
we generally use only a single set of `use` statements so they're all sorted together.

We also tend to group imports by package like `use hyper::{server::accept, service::make_service_fn, Server}`

Reply:
it was too much to hope that `cargo fmt` would take care of this for me

Reply:
@adleong it will sort them within blocks, but by default it honors empty line separators (in case you want to separate them). I'm unsure if there's linter option to control this. Could be.